Update maintain-polkadot-parameters.md

DrW3RK opened the pull request:
As this is my first update to the Wiki, I am making sure I get the convention right

Do we link to referendums and include dates when the changes happen? 

Also, is this the correct location where we update the details or should we update this in FAQ page? Thanks!

DrW3RK committed:
Update maintain-polkadot-parameters.md

As this is my first update to the Wiki, I am making sure I get the convention right

Do we link to referendums and include dates when the changes happen? 

Also, is this the correct location where we update the details or should we update this in FAQ page? Thanks!

salmad3 reviewed (commented):
Thanks @DrW3RK, would also add it on the [Nominator](https://wiki.polkadot.network/docs/learn-nominator/#required-minimum-stake) page. A similar issue was opened over the weekend: https://github.com/w3f/polkadot-wiki/issues/2282

I'd say it's a good idea to add some info on the FAQ page - especially since this update aims to be temporary. There have also been many questions/concerns about it. In this case, it could be something along of the lines of "Why did the minimum stake change?" "Until When?", "What's Next?" etc... 

I don't believe we link or include dates when changes happen. 
 
It is also good practice to reference the issue if the PR (or commit) aims to address it (generally speaking). Here's some [GitHub Docs](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue) for reference.

DrW3RK committed:
Instructions pointing to chain state

DrW3RK committed:
point to polkadot.js for minimum bonding parameter

From the suggestions provided in #2282 I removed the hyperlinks that point to the scripts that my be outdated and instead, pointed it to chain state available through UI

DrW3RK commented:
@DannyS03 Thank you very much for all the suggestions above. I believe this takes care of the issues linked to this PR for the time being. I will work on implementing dynamic inserts through a new PR. If everything looks good, this can be merged with Master.

DrW3RK committed:
Update learn-nominator.md

min staking parameter may not change every era
